taskslist/db/person_seeds.rb

```diff
@@ -0,0 +1,10 @@
+people = [
```

Comment:
Because this file is `db/person_seeds.rb` you might need to make sure it is `require`'d inside of the `db/seeds.rb` file, otherwise I don't think it will automatically be run by `rake db:seed`.